stories/transitions/Bootstrap.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import style from 'dom-helpers/style';
+import style from 'dom-helpers/css';
```

Comment:
we should import this as `css`, then?

Reply:
Hi @taion, I'm not sure. The dom-helpers package exports something called `style` from its index.js file, so that's what you would need to import if you were using the whole package instead of just the one function. But that function is being imported from `css.ts`, which is what causes this issue here. I opened an issue on the dom-helpers repo since it's unclear what the API is supposed to look like.

Reply:
@jquense ?

Reply:
it's just a naming oddity, I don't  really care which one we go with. `css` is more jquery-y but not really correct (and conflicts with styled-component type libraries)